### PR TITLE
All around merging all existing PRs into one. (CREDIT GIVEN, IF YOU WOULD LIKE YOUR PR TO BE REMOVED FROM HERE LEAVE A COMMENT)

### PR DESCRIPTION
**Pull request by AToska21**
I added a lot of stuff.

### CODE REVIEW

File: docs/setup/SYSTEMD.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+WorkingDirectory=<fosscord directory>
+ExecStart=npm run start:bundle
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+```
+If you set it up correctly, you should be able to run ``systemctl start fosscord`` and, your fosscord works!
+
+### Distros without "systemd"
```
Comment (TheArcaneBrony) [inline docs/setup/SYSTEMD.md:35]: Most distros without systemd use a similar system to sysinitv, which uses .sh files for managing services

File: docs/setup/SYSTEMD.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+## Setup
+
+### Explaination
+
+A systemd service, otherwise known as a daemon, is when you use the incuded systemd/systemctl package with your distro
+to make the computer run your program on it's own, without you having to use screen or anything else.
+
+To do this, you need a medium understanding of linux.
+
+### Using systemctl, service or systemd
```
Comment (TheArcaneBrony) [inline docs/setup/SYSTEMD.md:12]: just call it systemd, and add `(Ubuntu, Debian, Arch, ...)`

File: docs/setup/SYSTEMD.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+[Service]
+User=root
+WorkingDirectory=<fosscord directory>
+ExecStart=npm run start:bundle
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+```
+If you set it up correctly, you should be able to run ``systemctl start fosscord`` and, your fosscord works!
```
Comment (TheArcaneBrony) [inline docs/setup/SYSTEMD.md:33]: Don't forget about `systemctl enable fosscord` as thisll make it start on boot, you can also do `systemctl enable --now fosscord` to enable and start in one command.

File: docs/setup/SYSTEMD.md (hunk 1 above)
Comment (AToska21) [reply docs/setup/SYSTEMD.md:35]: you can install systemctl on other linux distros.

File: docs/setup/SYSTEMD.md (hunk 2 above)
Comment (AToska21) [reply docs/setup/SYSTEMD.md]: did that

File: docs/setup/SYSTEMD.md (hunk 3 above)
Comment (AToska21) [reply docs/setup/SYSTEMD.md]: yep! done that!

**Comment by AToska21** [comment]
ready to be merged!

**Pull request merged**